Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/CapsuleViewportEdit.h

```diff
@@ -52,18 +51,21 @@ namespace AzToolsFramework
             const AZ::Transform& worldTransform,
             const AZ::Transform& localTransform,
             const AZ::Vector3& nonUniformScale);
-        void SetupHeightManipulator(
+        AZStd::shared_ptr<LinearManipulator> SetupHeightManipulator(
```

Comment:
suggestion: Maybe now this is returning a value we should rename to `Create...` instead of `Setup`?

Reply:
I think it might be more important if it was part of a public API, but I don't see it as super critical here.

Reply:
Okay, I guess renaming is simpler if it's not public too (as there's no risk of breaking someone) but it's not super important